static/app/components/replays/breadcrumbs/replayTimelineEvents.tsx

```diff
@@ -105,16 +107,41 @@ function Event({
   // We just need to stack up to 3 times
   const totalStackNumber = Math.min(crumbs.length, 3);
 
+  // If there is only 1 event use the tab navigation handler on the node
+  const nodeClickHandler = () => {
+    if (crumbs.length === 1) {
```

Comment:
oh, i didn't think about this case. sucks to have to deal with this, hrm

Reply:
Just let me know what you think. It works fine but also may be a little bit confusing to users